Justfile

```diff
@@ -299,13 +299,21 @@ debug-info:
 # Update CONTRIBUTORS.md file
 [group('build')]
 @contributors:
-update-contributors:
     echo "Updating CONTRIBUTORS.md..."
-    # Cross-platform way to update CONTRIBUTORS.md using git shortlog
-    echo "# Contributors" > CONTRIBUTORS.md
-    echo "" >> CONTRIBUTORS.md
-    git shortlog -sne >> CONTRIBUTORS.md
-    echo "✓ Contributors list updated"
+    if [ "$$OS" = "Windows_NT" ]; then \
```

Comment:
This seems unrelated, if on purpose please put in a separate PR or justify why it should be done here.